src/NLog/LogFactory.cs

```diff
@@ -338,7 +338,7 @@ public LoggingConfiguration Configuration
                         }
                     }
 
-                    this.OnConfigurationChanged(new LoggingConfigurationChangedEventArgs(value, oldConfig));
+                    this.OnConfigurationChanged(new LoggingConfigurationChangedEventArgs(oldConfig, value));
```

Comment:
If I call `new LoggingConfigurationChangedEventArgs(value, oldConfig)` after this PR has been merged, then it has a different behavior? Or am I missing something :)